docs/uptime-guide/deployment-arch.asciidoc

```diff
@@ -0,0 +1,13 @@
+[role="xpack"]
+[[uptime-deployment-arch]]
+== Deployment Architecture
+
+There are infinite ways to deploy Uptime and Heartbeat, some of which work better than others. A guiding principle in these recommendations is that an outage that takes down the service being monitored should not also take down Heartbeat. You will want Heartbeat to be functioning even when your site is not, so the guidelines that follow attempt to minimize this possibility.
+
+Heartbeat is generally run as a centralized service within a datacenter. While it is possible to run it as a separate "sidecar" process paired with each process/container we recommend against it. Running Heartbeat centrally ensures you will still be able to see monitoring data in the event of an overloaded, disconnected, or otherwise malfunctioning server. 
+
+For further redundancy, you may want to deploy multiple Heartbeats across geographic and/or network boundaries to provide more data. Specify Heartbeat's observer https://www.elastic.co/guide/en/beats/heartbeat/current/configuration-observer-options.html[geo options] to do so. Some examples might be:
```

Comment:
When you fix the link, can you check the exact wording in the area of the link? As written, I think it's going to read "Specify Heartbeat's observer geo options" where "geo options" is the link. The heading on the linked page is "Specify observer and geo options". If you want to refer the reader to just the geo options, then you should probably drop "observer" in the text here. If it's both, then the linked text should probably be "observer and geo options".